docroot/sites/classrooms.uiowa.edu/modules/classrooms_core/classrooms_core.module

```diff
@@ -171,11 +171,27 @@ function classrooms_core_preprocess_block(&$variables) {
   }
 }
 
+/**
+ * Implements hook_preprocess_file_link().
+ */
+function classrooms_core_preprocess_file_link(&$variables) {
+  unset($variables['attributes']['class']);
+}
+
 /**
  * Implements hook_preprocess_field().
  */
 function classrooms_core_preprocess_field(&$variables, $hook) {
   switch ($variables['element']['#field_name']) {
+    case 'field_room_seating_chart':
+      if (!empty($variables['items'][0]['content']['#file'])) {
+        $new_filename = 'Seating Chart';
+        $file = $variables['items'][0]['content']['#file'];
+        $file->setFilename($new_filename);
+        $file->save();
+      }
```

Comment:
I think this needs to be done in a pre-save hook rather than a preprocess hook.

Reply:
Also, can this not be populated via defaults?

Reply:
I did not see a way for this to be populated via defaults.  I'm only seeing the option to provide a description text field.

Reply:
I'm not seeing a better way to do this. My other idea was to enable the description field, the populate it and disable it in the form alter. But it seems like that is probably about the same code footprint.